travis-ci/Dockerfile

```diff
@@ -9,14 +9,14 @@ RUN apt-get update --fix-missing && apt-get install -y wget bzip2 ca-certificate
     git mercurial subversion
 
 RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh && \
-    /bin/bash /Miniconda2-latest-Linux-x86_64.sh -b -p /opt/conda && \
-    rm Miniconda2-latest-Linux-x86_64.sh
+    wget --quiet https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    /bin/bash /Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda && \
+    rm Miniconda3-latest-Linux-x86_64.sh
 
 ENV PATH /opt/conda/bin:$PATH
 
 RUN conda config --set show_channel_urls true && conda update --yes conda && \
-    conda install --yes conda-build=1.21 jinja2 anaconda-client && conda config --add channels conda-forge --add channels svidela
+    conda install --yes conda-build jinja2 anaconda-client && conda config --add channels conda-forge --add channels svidela
 
```

Comment:
I remember there was an issue with `conda-build>1.21` related to the function `bldpkg_path` used in [cp_artifacts.py](https://github.com/bioasp/caspo/blob/master/travis-ci/cp_artifacts.py). See this [commit](https://github.com/bioasp/caspo/commit/e182e627a244a90f7109599ccf82d742a8fbdb15) (not very informative anyways :))

Reply:
Yes, travis is failing because of that, but conda-build 1.21 is no longer available apparently.
What is is the purpose of this artifact thing?

Reply:
The artifact is the conda package. After it's build in travis, I needed to copy it to make the upload to Anaconda Cloud

Reply:
Looking at the travis log, just found this at the end:

```
# Automatic uploading is disabled
# If you want to upload package(s) to anaconda.org later, type:
anaconda upload /opt/conda/conda-bld/linux-64/caspo-3.0.2.dev0-py37_0.tar.bz2
# To have conda build upload to anaconda.org automatically, use
# $ conda config --set anaconda_upload yes
anaconda_upload is not set.  Not uploading wheels: []
```

I don't remember to have that option 4 years ago. You may want to try that out and get rid of [upload.py](https://github.com/bioasp/caspo/blob/master/travis-ci/upload.py) and [cp_artifacts.py](https://github.com/bioasp/caspo/blob/master/travis-ci/cp_artifacts.py)

Reply:
Will try that. Btw, why are the packages architecture dependent? There is only python code, no?

Reply:
fixing the recipe fixed the `cp_artifact` (https://github.com/bioasp/caspo/pull/18/commits/cd49d85935183ffad807c85c358d2930a2b25041), but the upload fails due to deprecated modules. The upload option of anaconda is the best way to go, but I need to figure out out to use the token and so on (I'm used to do the anaconda upload using github actions, not travis,..); note that the recipe specify noarch (previous comment), so the upload should be done only once (e.g., on linux)

Reply:
Yes, python code only. But I think I had to make arch dependent releases in order to let conda find the right clingo release in conda.

Reply:
I'm fine with switching to Github actions if you prefer it over Travis.

Reply:
ok, so this is probably fine now, according to travis logs; I'll fix the upload later

Reply:
Yes, I'll probably switch to github actions; let's keep that for another PR :-)